NewStarterProject/Controllers/DoorController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NewStarterProject.Dtos;
+using NewStarterProject.Model;
+using NewStarterProject.Services;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace NewStarterProject.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DoorController : ControllerBase
+    {
+
+        private DoorService _service;
+
+        public DoorController(StarterProjectContext context)
+        {
+            _service = new DoorService(context);
+        }
+
+        
+        // POST api/<DoorController>
+        [HttpPost]
+        public async Task<ActionResult> Post([FromBody] DoorDto doorDto)
+        {
+            return Ok(await _service.CreateOrUpdateDoor(doorDto));
```

Comment:
It would be great to name this similarly to how you have done in the other controllers (_doorService). 
Being specific with variable names really helps us in a large code base like P10 especially since we tend to do a Search All every now and then to see if a particular service is used accross multiple controllers or other places in the code base.